contributing.md

```diff
@@ -28,32 +33,31 @@ After you do the changes, run:
 
     check
 
-It will run all the code checks and the unit tests making sure you
-won't break the build when your changes are pushed. If the checks
-doesn't succeed because of coding style, see the coding style guide
-[for web](web-style.md.html) or [for Python](python-style.md.html).
-If a test doesn't pass either your code needs to be fixed or the test
-need to be adapted.
+It will run all the code checks and the unit tests making sure you won't break the build when your changes are pushed. 
+If the checks doesn't succeed because of coding style, see the coding style guide [for web](web-style.md.html) or [for Python](python-style.md.html). 
+If a test doesn't pass either your code needs to be fixed or the test need to be adapted.
+
+### Open an Issue
+
+We track issues in http://bugs.sugarlabs.org
```

Comment:
This PR updates the markup but doesn't substantially change the content. 

On this topic, as you know, I think we should, but I don't feel there is sufficient clarify on the mailing list to proceed migrating it myself. 

If you think we should, I'd be grateful if you could voice your opinion on the mailing list thread :)